[FIX] spreadsheet_dashboard_account: invoicing dashboard consider out_refund

ppat-odoo opened the pull request:
Problem:- In Invoicing Dashboard, in some parts we get wrong values because of
wrongly set domain making dashboard consider Vendor Credit Note instead of
Customer Credit Note.

Before this commit:-
- In Invoicing Dashboard, reports Top Countries, Top Categories, Top Products,
  and Top Salespeople considers Customer Invoice and Vendor Credit Note.
- When clicked on Top Invoices, moves in draft and cancel state are also
  displayed.

After this commit:-
- In Invoicing Dashboard, reports Top Countries, Top Categories, Top Products,
  and Top Salespeople considers Customer Invoice and Customer Credit Note only.
- When clicked on Top Invoices, moves in draft and cancel state are not
  displayed.

task-4851920

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/213723.png)](https://mergebot.odoo.com/odoo/odoo/pull/213723)

ppat-odoo commented:
Hello @SaddemAmine , thanks for the review.
As you said we don't want an inline comment so I have updated commit message and PR description with explanation why we switched from price_subtotal to move_type.
Can you please have a look if it explains clearly?

malb-odoo commented:
@ppat-odoo usually we try to avoid unrelated change because it will break the github history, specially here when it's working and it's only to have some consistency. I would remove it :smile:

malb-odoo commented:
@robodoo r+
